Gopkg.toml

```diff
@@ -143,3 +143,7 @@
 [[constraint]]
   name = "github.com/aliyun/alibaba-cloud-sdk-go"
   version = "1.27.7"
+
+[[override]]
+  name = "github.com/json-iterator/go"
+  version = "v1.1.5"
```

Comment:
Override the version due to issue: https://github.com/kubernetes/apimachinery/issues/46